isilon_create_users.sh

```diff
@@ -213,125 +213,123 @@ echo "# use this file to add to the group file of your clients" | cat > $grpfile
 done
 
 # Special cases
 case "$DIST" in
-    "cdh")
-        #Deal with special cases on Isilon 
-	isi auth groups modify sqoop$CLUSTER_NAME --add-user sqoop2$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user sqoop2$CLUSTER_NAME to sqoop$CLUSTER_NAME group in zone $ZONE"
-        isi auth groups modify hive$CLUSTER_NAME --add-user impala$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user implala$CLUSTER_NAME to hive$CLUSTER_NAME group in zone $ZONE"
-	isi auth groups modify hadoop$CLUSTER_NAME --add-user hbase$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user hbase$CLUSTER_NAME to hadoop$CLUSTER_NAME group in zone $ZONE"
-	
-	#Set some varliables to take these special case group assignments to the group file
-        sqp=`grep sqoop$CLUSTER_NAME $grpfile`
-        hve=`grep hive$CLUSTER_NAME $grpfile`
-
-	#manipulate the group file for special cases
-        sed -i .bak /$sqp/d $grpfile
-        echo "$sqp,sqoop2" | cat >> $grpfile
-        [ $? -ne 0 ] && addError "Could not add user sqoop2$CLUSTER_NAME to sqoop$CLUSTER_NAME group in $grpfile"
-        sed -i .bak /$hve/d $grpfile
-        echo "$hve,impala" | cat >> $grpfile
-        [ $? -ne 0 ] && addError "Could not add user impala$CLUSTER_NAME to hive$CLUSTER_NAME group in $grpfile"
-        ;;
-    "bi")
-        isi auth groups modify users$CLUSTER_NAME --add-user hive$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user hive$CLUSTER_NAME to users$CLUSTER_NAME group in zone $ZONE"
-        isi auth groups modify hcat$CLUSTER_NAME --add-user hive$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user hive$CLUSTER_NAME to hcat$CLUSTER_NAME group in zone $ZONE"
-        hct=`grep hcat$CLUSTER_NAME: $grpfile`
-        sed -i .bak /$hct/d $grpfile
-        echo "$hct,hive$CLUSTER_NAME" | cat >> $grpfile
-        [ $? -ne 0 ] && addError "Could not add user hive$CLUSTER_NAME to hcat$CLUSTER_NAME group in $grpfile"
-        isi auth groups modify knox$CLUSTER_NAME --add-user kafka$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user kafka$CLUSTER_NAME to knox$CLUSTER_NAME group in zone $ZONE"
-        knx=`grep knox$CLUSTER_NAME: $grpfile`
-        sed -i .bak /$knx/d $grpfile
-        echo "$knx,kafka$CLUSTER_NAME" | cat >> $grpfile
-        [ $? -ne 0 ] && addError "Could not add user kafka$CLUSTER_NAME to knox$CLUSTER_NAME group in $grpfile"
-        ;;
+    "cdh")
+        #Deal with special cases on Isilon
+        isi auth groups modify sqoop$CLUSTER_NAME --add-user sqoop2$CLUSTER_NAME --zone $ZONE
+        [ $? -ne 0 ] && addError "Could not add user sqoop2$CLUSTER_NAME to sqoop$CLUSTER_NAME group in zone $ZONE"
+        isi auth groups modify hive$CLUSTER_NAME --add-user impala$CLUSTER_NAME --zone $ZONE
+        [ $? -ne 0 ] && addError "Could not add user implala$CLUSTER_NAME to hive$CLUSTER_NAME group in zone $ZONE"
```

Comment:
I actually went ahead an fixed this. It could see leaving it if it matched the user being added just above, but as it is, this is just telling the user one thing while doing another (clearly due to a typo).